whetstone/compiler/src/main/kotlin/com/freeletics/mad/whetstone/codegen/common/ComposeGenerator.kt

```diff
@@ -21,24 +20,30 @@ internal class ComposeGenerator(
 ) : Generator<ComposeData>() {
 
     internal fun generate(): FunSpec {
+        val parameterString = data.composableParameter
+            .joinToString { "${it.name} = ${it.className.propertyName}" }
+            .let { if(it.isNotBlank()) it.plus(", ") else it }
```

Comment:
I think it would make sense to build the whole composable call as a `CodeBlock` here and also make `state = currentState` and `sendAction = { ... }` named parameters

Reply:
Yes